csharp/Microsoft.Azure.Databricks.Client/Run.cs

```diff
@@ -100,5 +100,29 @@ public class Run : RunIdentifier
         /// </summary>
         [JsonProperty(PropertyName = "run_page_url")]
         public string RunPageUrl { get; set; }
+
+        /// <summary>
+        /// The time at which this run was finished in epoch milliseconds (milliseconds since 1/1/1970 UTC).
+        /// </summary>
+        [JsonProperty(PropertyName = "start_time")]
+        [JsonConverter(typeof(MillisecondEpochDateTimeConverter))]
+        public DateTimeOffset? EndTime
+        {
+            get
+            {
+                if (StartTime.HasValue == false)
+                {
+                    return null;
+                }
+
+                var setupDuration = TimeSpan.FromMilliseconds(SetupDuration);
+                var cleanupDuration = TimeSpan.FromMilliseconds(CleanupDuration);
+                var executionDuration = TimeSpan.FromMilliseconds(ExecutionDuration);
+
+                var jobExecution = setupDuration.Add(cleanupDuration).Add(executionDuration);
+
+                return StartTime.Value.Add(jobExecution);
```

Comment:
We should also check for the run state - if the run is still executing, the EndTime field should be left null since the run hasn't ended yet.

Reply:
That's fine for me, would it be ok if I add an `IsCompleted` property that surfaces this based on `State` as well?

Reply:
yeah that sounds good.

Reply:
Done!